Add active_series_additional_custom_trackers config option (#10428)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

# CLI flag: -ingester.ooo-native-histograms-ingestion-enabled
[ooo_native_histograms_ingestion_enabled: <boolean> | default = false]

# (advanced) Custom trackers for active metrics. If there are active series

> # matching a provided matcher (map value), the count will be exposed in the

Reviewer: ```suggestion
# matching a provided matcher (map value), the count is exposed in the
```

Author: Addressed in https://github.com/grafana/mimir/pull/10428/commits/b66faea0d175b7d3cbe72bb40f014b00b52df972

> # custom trackers metric labeled using the tracker name (map key). Zero valued
> # counts are not exposed (and removed when they go back to zero).

Reviewer: ```suggestion
# custom trackers metric labeled using the tracker name (map key). Zero-valued
# counts are not exposed and are removed when they go back to zero.
```

Author: Addressed in https://github.com/grafana/mimir/pull/10428/commits/b66faea0d175b7d3cbe72bb40f014b00b52df972

# CLI flag: -ingester.active-series-custom-trackers
[active_series_custom_trackers: <map of tracker name (string) to matcher (string)> | default = ]

# (advanced) Additional custom trackers for active metrics merged on top of the

> # custom trackers. This configuration option can be useful to define the base

Reviewer: Does `merged on top of the custom trackers` refer to preexisting trackers? If so, we should specify this for clarity.

Reviewer: Reading the rest of this, it looks like that are merged on top of base custom trackers? I think we should say that.

Author: Yes your understanding is correct. I've addressed it in https://github.com/grafana/mimir/pull/10428/commits/b66faea0d175b7d3cbe72bb40f014b00b52df972

> # custom trackers. This configuration option can be useful to define the base
> # custom trackers globally for all tenants, and use the additional trackers to
> # add extra trackers on a per-tenant basis.

Reviewer: ```suggestion
# custom trackers. You can use this configuration option to define the base
# custom trackers globally for all tenants, and then use the additional trackers to
# add extra trackers on a per-tenant basis.
```

Author: Addressed in https://github.com/grafana/mimir/pull/10428/commits/b66faea0d175b7d3cbe72bb40f014b00b52df972

# Example:

> #   The following configuration will count the active series coming from dev and

Reviewer: ```suggestion
#   The following configuration counts the active series coming from dev and
```

Author: Addressed in https://github.com/grafana/mimir/pull/10428/commits/b66faea0d175b7d3cbe72bb40f014b00b52df972

> #   prod namespaces for each tenant and label them as {name="dev"} and

Reviewer: ```suggestion
#   prod namespaces for each tenant and labels them as {name="dev"} and
```

Author: Addressed in https://github.com/grafana/mimir/pull/10428/commits/b66faea0d175b7d3cbe72bb40f014b00b52df972
